streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/controller/PassportController.java

```diff
@@ -114,4 +115,9 @@ private RestResponse login(String username, String password, User user) throws E
         userService.generateFrontendUserInfo(user, user.getLastTeamId(), jwtToken);
     return new RestResponse().data(userInfo);
   }
+
+  @PostMapping("types")
+  public RestResponse userTypes() {
```

Comment:
I don't think we need this method

Reply:
done. btw, refer to [UserController.java#L158](https://github.com/apache/incubator-streampark/blob/dev/streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/controller/UserController.java#L158), we may alse need to refactor it.